lib/Infer/ExhaustiveSynthesis.cpp

```diff
@@ -322,71 +326,82 @@ void getGuesses(std::vector<Inst *> &Guesses,
   // need two reserved per select instruction
   Inst *C2 = IC.getReservedConst();
   Comps.push_back(C2);
+  Inst *C3 = IC.getReservedConst();
```

Comment:
Why create a new constant here, we want to prune (ter-op c,c,c) as early as possible.

Reply:
I see, ignore this comment